docs/mautic_community_portal/signing_in_and_configuring_your_profile.rst

```diff
@@ -0,0 +1,251 @@
+Check the dropdown menu item underneath your name. This allows you to customize your account, your public profile, and also see notifications and direct messages. If you're an administrator, you access the administrator portal here, too.
+
+.. vale off
+
+My Account
+==========
+
+This is where you can configure your avatar that others see in the portal whenever you engage here. You can configure your email address and all of your details and which locale you prefer to browse the website using.
+
+.. vale on
+
+Notifications
+=============
+The notification settings is important. This allows you to actually control what you get notified about within the community and how frequently.
+
+.. image:: images/notifications.png
+    :alt: VSCode screenshot showing how to change branches
+    :width: 600px
+    :align: center
+
+.. vale off
+
+You can select or deselect the things that you would like to receive notifications about. You can choose if you get a daily summary email, a weekly summary email, real time or no updates by email. Daily or weekly would be a good option.
+
+.. vale on
+
+Updates
+========
+
+.. vale off
+
+Updates is a bit like a newsletter feature. The community may send an update every now and then by email. If you want to receive those, then you can select that here.
+
+.. vale on
+
+Direct messages
+===============
+Do you want to allow anyone to send you a direct message? This allows anyone to send you a direct message on this platform, even if you don't follow them.
+
+Deselect the option if you only want people that you follow to send you a direct message.
+
+.. vale off
+
+My Interests
+============
+
+.. vale on
+
+This allows you to decide what topics you'd like to see in your timeline. It's recommended to leave them all selected and if it becomes too noisy you can turn them off in the future.
+
+.. vale off
+
+My Data
+=======
+The 'My Data' section allows you to download all the information held on you in this system, and to delete your account.
+
+My Public Profile
+******************
+
+Under 'My Public Profile' in the top user menu, this is what your profile looks like to the public.
+
+.. vale on
+
+Timeline
+========
+
+.. vale off
+
+This shows all of the ways in which you have engaged across the portal by time, with a dropdown filter that allows you to only show certain types of activity. This section **isn't** visible to any other user than the logged in user themselves, because it can also contain items that should be visible only to that participant.
+
+.. vale on
+
+Activity
+========
+
+.. image:: images/activity.png
+    :alt: VSCode screenshot showing how to change branches
+    :width: 600px
+    :align: center
+
+.. raw:: html
+
+    <br><br>
+
+Activity is showing the activity that you're involved in. This might include endorsing/supporting proposals, commenting on/creating resources, and so forth. Again it's filterable by type, but this shows on your public profile and other people can see it.
```

Comment:
```suggestion
Activity is showing the activity that you're involved in. This might include liking/voting on proposals, commenting on/creating resources, and so forth. Again it's filterable by type, but this shows on your public profile and other people can see it.
```